frontend/src/app/app-routing.module.ts

```diff
@@ -37,6 +37,7 @@ const appRoutes: Routes = [
   {
     path: '',
     data: {
+      title: 'US Forest Service ePermit',
```

Comment:
does this end up being a prefix? or does it get overwritten by individual pages?

Also we may want to update this here to 'US Forest Service Open Forest' now.

Reply:
It should get overwritten by the child pages. I'll update the name to Open Forest 👍